### PR TITLE
FSET-1100: Reminder for online tests (3 and 1 day before expiration)

### PR DESCRIPTION
**Pull request by Lansalo**

### CODE REVIEW

File: app/services/onlinetesting/OnlineTestExpiryService.scala [hunk 1]
```diff
@@ -79,4 +95,13 @@ class OnlineTestExpiryServiceImpl(
       Map("userId" -> expiringTest.userId) ++ emailAddress.map("email" -> _).toMap
     )
   }
+
+  private def auditF(event: String, userId: String, emailAddress: Option[String] = None): Future[Unit] = {
+    // Only log user ID (not email).
```
Comment (kjharkin) [inline app/services/onlinetesting/OnlineTestExpiryService.scala]: Looks like you're logging the email address despite the comment

File: app/model/ProgressStatuses.scala [hunk 2]
```diff
@@ -42,6 +42,8 @@ object ProgressStatuses extends Enum {
 
   val OnlineTestInvitedProgress = EnumVal(ApplicationStatuses.OnlineTestInvited.name.toLowerCase, 110)
   val OnlineTestStartedProgress = EnumVal(ApplicationStatuses.OnlineTestStarted.name.toLowerCase, 120)
+  val OnlineTestFirstExpiryNotification = EnumVal("test_expiry_first_reminder", 121)
+  val OnlineTestSecondExpiryNotification = EnumVal("test_expiry_second_reminder", 122)
```
Comment (kjharkin) [inline app/model/ProgressStatuses.scala]: Can we create a constant for "test_expiry_first_reminder" and "test_expiry_second_reminder"?

File: app/services/onlinetesting/OnlineTestExpiryService.scala (hunk 1 above)
Comment (Lansalo) [inline app/services/onlinetesting/OnlineTestExpiryService.scala]: Well spotted. Problem is that mine is a Future version of the existing "audit" method which has the same issue. Either we get rid of the comment or we stop logging the email

File: app/model/ProgressStatuses.scala (hunk 2 above)
Comment (Lansalo) [inline app/model/ProgressStatuses.scala]: Is already part of a constant value (val in an object). Is not easy to see it from the review, but we already use the same syntax (see lines 31-40 for example).

Comment (kjharkin) [inline app/model/ProgressStatuses.scala]: I think you may have misunderstood me. I don't want literal strings in the code - where we find them we should actively remove them. 
e.g
val TestExpiryFirstReminder = "test_expiry_first_reminder"

scope as you see fit.

**Pull request merged**